### PR TITLE
Deploy style guide with TravisCI 📦

### PR DESCRIPTION
**Pull request by gnclmorais**
### ~Review after https://github.com/BridgeU/bridget/pull/12~

This pull request allow us to automatically deploy Bridget into a `gh-pages` branch that GitHub will automatically use to present our style guide at http://bridgeu.github.io/bridget.

It’s easy for us developers to download GitHub projects into our machines, install the missing dependencies and simply run them. However, some people might not have any of these tools installed on their computers and simply want to see our visual language. In this case, having an easily accessible project like this is valuable.

Example of successful build (https://travis-ci.org/gnclmorais/bridget) and what it would look like under the BridgeU repository (https://gnclmorais.github.io/bridget). 📄

---

### Why TravisCI instead of CircleCI?
- Deploying GitHub Pages with [TravisCI][travis-deply] seems simpler that with [CircleCI][circle-deploy]
- Bridget is public (at the moment) so TravisCI use is free
- Moving this to TravisCI would mean one less project cluttering our already (sometimes) struggling CircleCI flow

---

More info:
- https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages
- https://docs.travis-ci.com/user/deployment/pages/
- https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#specifying-nodejs-versions-using-nvmrc

[travis-deply]: https://docs.travis-ci.com/user/deployment/pages/
[circle-deploy]: https://circleci.com/blog/deploying-documentation-to-github-pages-with-continuous-integration/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - node_modules
+
+before_deploy:
+  - npm install
+  - npm run build-app
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
```
Comment (gnclmorais) [inline .travis.yml]: We need to generate this and feed it into TravisCI.

**Comment by carlosipe** [comment]
Just wondering why Travis and not CircleCI? I really TravisCI, it's not an objection, just curiosity.

**Comment by gnclmorais** [comment]
Good question, I added that to the [PR description](https://github.com/BridgeU/bridget/pull/13#issue-274336420). ✍️

**Comment by gnclmorais** [comment]
Exactly!

**Comment by gnclmorais** [comment]
Marking this as *`standby`* so I can talk with Gareth first about BridgeU @ TravisCI.

**Pull request merged**